src/appsettings/__init__.py

```diff
@@ -197,6 +201,34 @@ def check(cls):
         if exceptions:
             raise ImproperlyConfigured("\n".join(exceptions))
 
+    def manage_environ_invalidation(self, signal, sender, setting, value, enter):
```

Comment:
Signal handler needs `**kwargs` according to documentation. That would also narrow down the unused arguments.

Reply:
Fixed.